### PR TITLE
Update app lifecycle troubleshooting doc in 2.4

### PR DESCRIPTION
**Pull request by xiangjingli**
https://github.com/open-cluster-management/backlog/issues/16822

### CODE REVIEW

File: troubleshooting/trouble_local_cluster.adoc [hunk 1]
```diff
@@ -1,17 +1,17 @@
 [#troubleshooting-local-cluster-not-selected]
 = Troubleshooting local cluster not selected with placement rule
 
-The managed clusters are selected with a placement rule, but the `local-cluster` (hub cluster that is also managed) is not selected. The placement rule user is not granted to permission to create deployable resources in the `local-cluster` namespace.
+The managed clusters are selected with a placement rule, but the `local-cluster` (hub cluster that is also managed) is not selected. The placement rule user is not granted permission to get the managedcluster resources in the `local-cluster` namespace.
```
Comment (dockerymick) [inline troubleshooting/trouble_local_cluster.adoc]: @cadawson whenever you get to this, I recommend the following change `managedcluster` resources

I can also update the file if you need some help w/ PRs

Comment (swopebe) [reply troubleshooting/trouble_local_cluster.adoc]: I will go ahead and take this one.

**Review by swopebe — APPROVED** [review]
approve after edits

**Comment by xiangjingli** [comment]
@swopebe  Thanks! The new updates looks good!

**Pull request merged**